weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/LLMDropdown.tsx

```diff
@@ -19,18 +19,24 @@ export const LLMDropdown: React.FC<LLMDropdownProps> = ({value, onChange}) => {
     // for each provider, get all the LLMs that are supported by that provider
     label: LLM_PROVIDER_LABELS[provider],
     // filtering to the LLMs that are supported by that provider
-    options: Object.keys(LLM_MAX_TOKENS).reduce<
-      Array<{group: string; label: string; value: string}>
-    >((acc, llm) => {
-      if (LLM_MAX_TOKENS[llm as LLMMaxTokensKey].provider === provider) {
-        acc.push({
-          group: provider,
-          label: llm,
-          value: llm,
-        });
-      }
-      return acc;
-    }, []),
+    options: Object.keys(LLM_MAX_TOKENS)
+      .reduce<Array<{group: string; label: string; value: string}>>(
+        (acc, llm) => {
+          if (LLM_MAX_TOKENS[llm as LLMMaxTokensKey].provider === provider) {
+            acc.push({
+              group: provider,
+              label:
+                llm.includes(provider) || provider === 'openai'
```

Comment:
TIOLI: i'm becoming a simp for micro functions, could be a nice place for one that explains why we do this logic

Reply:
like: 
```javascript
function _make_dropdown_label(provider: string): string {
```